tests/test_modeling_flax_bert.py

```diff
@@ -105,7 +105,26 @@ def prepare_config_and_inputs_for_common(self):
 @require_flax
 class FlaxBertModelTest(FlaxModelTesterMixin, unittest.TestCase):
 
-    all_model_classes = (FlaxBertModel,) if is_flax_available() else ()
+    all_model_classes = (FlaxBertModel, FlaxBertForMaskedLM) if is_flax_available() else ()
 
     def setUp(self):
         self.model_tester = FlaxBertModelTester(self)
+
+    def test_naming_convention(self):
+        for model_class in self.all_model_classes:
+            model_class_name = model_class.__name__
+            module_class_name = (
+                model_class_name[:-5] + "Module" if model_class_name[-5:] == "Model" else model_class_name + "Module"
+            )
+            bert_modeling_flax_module = __import__(
+                "transformers.models.bert.modeling_flax_bert", fromlist=[module_class_name]
```

Comment:
I think this can be replaced by `model_class.__module__` which then means you can put this in the common tests.